README.md

```diff
@@ -26,6 +26,44 @@ add as much states as preferred, and to create custom natural history of the dis
 
 For more information, please refer to the official documentation of the project.
 
+## Testing
+
+All the test were developed with the python tool pytest. To see them properly, type 
+in the command line: 
+
+`$ pytest -s -v <name_of_test>`
+
+During test execution with pytest, any output sent to stdout and stderr is 
+[captured](https://docs.pytest.org/en/6.2.x/capture.html). the `-s` flag  
+disable all capturing and let the user look at all the outputs, this is very 
+important because the description of all test were typed in the docstring of each method.
+
+The `-v` flag indicate to pytest run in [verbosity](https://docs.pytest.org/en/latest/how-to/output.html) 
+mode, this control all the pytest outputs.
+
+If you want to run all the test at the same time, you must be inside of `abmodel/tests`, 
+and type in the command line:
+
+`$ pytest -s -v`
```

Comment:
Change to use code block syntasis. See https://www.freecodecamp.org/news/how-to-format-code-in-markdown/